.github/workflows/docusaurus.yml

```diff
@@ -7,7 +7,7 @@ concurrency:
 on:
   push:
     branches:
-      - main
+      - develop
```

Comment:
@aabedraba should we publish changes for develop branch?

Reply:
@ypoplavs correct